app/views/activity.scala

```diff
@@ -259,11 +259,11 @@ object activity {
       )
     )
 
-  private def renderStream(u: User) =
+  private def renderStream(u: User)(implicit ctx: Context) =
     entryTag(
       iconTag(""),
-      "Hosted a ",
-      a(href := routes.Streamer.show(u.username))("live stream")
+      trans.activity.`hosted`()+" ",
```

Comment:
If the name is a valid scala identifier, there is no need to use ``. Use placeholders like %s in the translation instead of string concatenation.

Reply:
OK, I refactored to `trans.activity.hostedX("")`. I passed an empty string because I wanted `a live stream` to be within the link, not in plain text.

Reply:
It should be possible to put the link into the placeholder: `trans.activity.hostedX(a(href := ...)(...))`

Reply:
OK, it compiles. As mentioned, not sure how to do a dev live stream, so didn't fully test this.

Reply:
I feel sorry to tell you that now, but I'd actually like if it were a single translation, fully included in the `<a>`. Would make it easier for translators to get it right.

Reply:
No problem, I actually considered doing it that way.